notebooks/scripts/loading_model.py

```diff
@@ -0,0 +1,88 @@
+import torch
+import os
+import sys
+from models.unet import UNet                   #UNet model. From: https://github.com/johschmidt42/PyTorch-2D-3D-UNet-Tutorial
+
+current_directory = os.getcwd()                #get current working directory
+    
+#importing HighResNet and VNetLight models from medzoopytorch library
+sys.path.append(current_directory + '/lib/medzoo')
```

Comment:
maybe not critical right now, but might be good to find a different install method here for an update.